BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -76,6 +81,15 @@ public class AdsViewController: UIViewController {
     adView.addGestureRecognizer(swipePanGesture)
   }
   
+  public override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
+    
+    if UIDevice.current.userInterfaceIdiom == .pad {
+      let isWidthLarger = view.bounds.width > view.bounds.height
+      widthAnchor?.constant = isWidthLarger ? view.bounds.width * 0.40 : view.bounds.height * 0.40
```

Comment:
We should probably round these to nearest valid pixel (0, 0.5, 1 on 2x devices, 0, 0.33, 0.66, 1 on 3x devices, though not sure if there's an iPad that supports that currently)